hurricane/src/isobar/meson.build

```diff
@@ -90,4 +90,13 @@ isobar = library(
   install: true,
 )
 
+py.extension_module(
+  'Hurricane',
+  isobar_sources,
+  dependencies: [qt_deps, py_deps, boost],
+  link_with: [isobar, hurricane],
+  include_directories: hurricane_includes,
+  install: true,
+  subdir: 'coriolis'
+)
```

Comment:
Hurricane extension is defined in the hurricane top level meson.build. Ok to move, but I didn't see a corresponding delete in this patch

Reply:
Oh. I didn't see it. I did expect the module to be built in the directory holding the files (like for ``viewer`` & ``analog`` (looks more "uniform" to me). Will remove it from top level. Strangely enough, I didn't see it in the ``site-packages/cotiolis/``, maybe I was too quick.

Reply:
As discussed, please leave at top level